
#header中时间是IMU硬件时间
Header header

float64 timeImuSys
float64 timeGpsUTC


